app/models/metric/rollup.rb

```diff
@@ -43,19 +43,22 @@ module Metric::Rollup
       :cpu_usage_rate_average,
       :derived_vm_numvcpus,
       :derived_memory_used,
-      :net_usage_rate_average
+      :net_usage_rate_average,
+      :derived_vm_allocated_disk_storage
     ],
     :ContainerService_container_groups    => [
       :cpu_usage_rate_average,
       :derived_vm_numvcpus,
       :derived_memory_used,
-      :net_usage_rate_average
+      :net_usage_rate_average,
+      :derived_vm_allocated_disk_storage
     ],
     :ContainerReplicator_container_groups => [
       :cpu_usage_rate_average,
       :derived_vm_numvcpus,
       :derived_memory_used,
-      :net_usage_rate_average
+      :net_usage_rate_average,
+      :derived_vm_allocated_disk_storage
```

Comment:
These lines here mean that `ContainerProject`, `ContainerService` and `ContainerReplicator` will also have allocated disk storage filled in through the container groups association. Im not sure thats relevant real-world information.